### PR TITLE
v4.0.1.

### PR DESCRIPTION
**Pull request by alexlamsl**
@XhmikosR PTAL

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,9 +23,9 @@
   },
   "devDependencies": {
     "grunt": "^1.0.3",
-    "grunt-contrib-clean": "^1.1.0",
+    "grunt-contrib-clean": "^2.0.0",
```
Comment (XhmikosR) [inline package.json]: The deps update shouldn't be in this commit. They are different changes.

**Comment by XhmikosR** [comment]
BTW, I removed you many months ago from the npm users from grunt-contrib-uglify, because you hadn't enabled 2FA. If you have enabled it, let me know and I will add you back.

File: package.json (hunk 1 above)
Comment (alexlamsl) [reply package.json]: So do we actually want to update these?

If so, will make a separate PR for this 👌

**Comment by alexlamsl** [comment]
I have 2FA for `npm` working now I think, i.e. every time I do `npm login` or `npm publish` I get prompted for "OTP token".

**Comment by XhmikosR** [comment]
@alexlamsl  can you rebase this and make sure package-lock.json doesn't have any unneeded stuff?

I added you back to npm BTW.

**Comment by alexlamsl** [comment]
Sorry for the delay - was in a call just now.

I've made sure `package-lock.json` only contains version string change.

**Comment by XhmikosR** [comment]
Let's wait for CI and then merge. Then create the GH release, which creates the tag and then npm publish.

**Comment by alexlamsl** [comment]
@XhmikosR heh, you beat me to tag and publish by mere seconds 👻

**Comment by XhmikosR** [comment]
haha, I wasn't sure if you were going to do it yourself :P

**Comment by alexlamsl** [comment]
AppVeyor build on `master` got stuck in the queue, so I was anxiously waiting for it to pass.

**Pull request merged**